lib/puppet/provider/selmodule/semodule.rb

```diff
@@ -118,11 +118,13 @@ def selmodversion_file
   end
 
   def selmodversion_loaded
-    lines = ()
```

Comment:
Is this what was causing that Rubocop rule to fail before?

Reply:
Yes that was the offending line.